api/aether-2.1.0-openapi3.yaml

```diff
@@ -201,12 +221,14 @@ components:
           title: application-id
           type: string
           x-go-type: ListKey
+          x-yang-type: application-id
```

Comment:
Because this new variable, the `Test_ValidateRequestGivesError` test is failing. Try to make sure that your changes do not break the code (and tests)
```diff
diff --git a/pkg/toplevel/server/validation_test.go b/pkg/toplevel/server/validation_test.go
index 60cb145..6dfc17b 100644
--- a/pkg/toplevel/server/validation_test.go
+++ b/pkg/toplevel/server/validation_test.go
@@ -65,7 +65,8 @@ Schema:
     "maxLength": 1024,
     "minLength": 1,
     "title": "description",
-    "type": "string"
+    "type": "string",
+    "x-yang-type": "description"
   }

 Value:
```

Reply:
done